src/config.ts

```diff
@@ -261,7 +261,7 @@ export const DEFAULT_APP_SETTINGS = {
   iconSize: iconSizeBias,
   navigationBarBehaviour: 'custom',
   searchEngine: SEARCH_ENGINE_STARTPAGE,
-  useVerticalStyle: false,
+  useHorizontalStyle: false,
```

Comment:
This will mean a breaking change for all pre-existing users - are we sure we want to rename? imo, its still ok to do so - coz "Ferdium" is still not released - but, we need to capture this in a release note.

Reply:
> This will mean a breaking change for all pre-existing users - are we sure we want to rename? imo, its still ok to do so - coz "Ferdium" is still not released - but, we need to capture this in a release note.

Yep, it is. I can revert it and leave it the other way around if needed. But we should change the code accordingly so that it makes sense.

Nevertheless, the change users will experience is that they need to toggle the toggle back again (I think that's it).

Reply:
yes - please call that out in the release notes when this gets released.